Mergin/project_status_dialog.py

```diff
@@ -68,6 +71,11 @@ def __init__(
         self.treeStatus.expandAll()
         self.changes_summary = push_changes_summary
 
+        try:
+            self.module = importlib.import_module("Mergin.repair")
+        except ImportError:
+            self.module = None
```

Comment:
when this can happen?

Reply:
In theory it should not happen. I can remove it if you think it is too much.

Reply:
+1 to do ordinary import at the top of the file to avoid confusion